dds/DCPS/XTypes/DynamicData.cpp

```diff
@@ -78,7 +99,6 @@ DynamicData::~DynamicData()
 
 void DynamicData::copy(const DynamicData& other)
 {
-  ACE_Message_Block::release(chain_);
```

Comment:
This needs to keep because we want to release the old message chain before storing a new chain into `chain_`.